vmm/src/cpu.rs

```diff
@@ -133,14 +151,18 @@ pub enum Error {
     #[error("Error initializing PMU: {0}")]
     InitPmu(#[source] hypervisor::HypervisorCpuError),
 
-    #[cfg(all(target_arch = "x86_64", feature = "gdb"))]
+    #[cfg(feature = "gdb")]
     #[error("Error during CPU debug: {0}")]
     CpuDebug(#[source] hypervisor::HypervisorCpuError),
 
     #[cfg(all(target_arch = "x86_64", feature = "gdb"))]
     #[error("Error translating virtual address: {0}")]
     TranslateVirtualAddress(#[source] hypervisor::HypervisorCpuError),
 
+    #[cfg(all(target_arch = "aarch64", feature = "gdb"))]
+    #[error("Error translating virtual address: {0}")]
+    TranslateVirtualAddress(#[source] anyhow::Error),
```

Comment:
This can be improved in a later PR.

Now x86 and aarch64 both return this error but their implementation is different.